README.md

```diff
@@ -67,7 +79,7 @@ The `profile:` section lists tuned profiles and their names.
 
       [sysctl]
```

Comment:
Is the [sysctl] part of the profile data optional? If the user wants to create a custom CR based on an existing tuned profile such as sap-hana, can they only specify the "include=sap-hana" directive under [main] and skip specifying the [sysctl] part?

Reply:
Yes, that's is correct.  The `[sysctl]` plugin is optional.